stdlib/universe/hour_selection_test.flux

```diff
@@ -65,3 +66,27 @@ testcase hour_selection_overnight_range {
 
         testing.diff(got: got, want: want)
     }
+
+testcase hour_selection_location {
+        option location = timezone.location(name: "America/Los_Angeles")
+
+        got =
+            csv.from(csv: inData)
+                |> range(start: 2018-12-01T00:00:00Z)
+                |> hourSelection(start: 9, stop: 10, timeColumn: "_time")
+
+        want =
+            csv.from(
+                csv:
+                    "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,string,string,dateTime:RFC3339,unsignedLong
+#group,false,false,true,true,true,true,false,false
+#default,_result,,,,,,,
+,result,table,_start,_stop,_measurement,_field,_time,_value
+,,0,2018-12-01T00:00:00Z,2030-01-01T00:00:00Z,Sgf,DlXwgrw,2018-12-18T17:11:25Z,33
+,,0,2018-12-01T00:00:00Z,2030-01-01T00:00:00Z,Sgf,DlXwgrw,2018-12-19T18:11:35Z,63
+",
+            )
+
+        testing.diff(got: got, want: want)
+    }
```

Comment:
This looks good. I checked that `2018-12-19T18:11:35Z` is within the 9-10 range in LA time, and it turns out that is 10:11 in LA time. But the semantics of `hourSelection` is that the `stop` hour is inclusive. This was a surprise to me! So this looks good.